cmd/migrate.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"go/types"
+	"strconv"
+
+	migrate "github.com/rubenv/sql-migrate"
+	"github.com/spf13/cobra"
+	"github.com/stellar/go/support/config"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/wallet-backend/internal/db"
+)
+
+type migrateCmd struct{}
+
+func (c *migrateCmd) Command() *cobra.Command {
+	var databaseURL string
```

Comment:
I'm seeing this config option repeating multiple times in the code perhaps we can make only this one a shared config.

Reply:
Agreed, done! ✅  [e6c6df9](https://github.com/stellar/wallet-backend/pull/18/commits/e6c6df99aefd4c5439ecaef1d3321c030b6a64ea)